README.md

```diff
@@ -422,9 +355,11 @@ A `PipelineRun` is how you can start a pipeline and tie it to the git and image
 
 ```bash
 $ tkn pipeline start build-and-deploy \
-    -r git-repo=api-repo \
-    -r image=api-image \
-    -p deployment-name=vote-api
+    -w name=shared-workspace,claimName=source-pvc \
+    -p deployment-name=vote-api \
+    -p git-url=http://github.com/openshift-pipelines/vote-api.git \
+    -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-api \
+    --showlog=true
```

Comment:
If `--showlog` flag is provided, the lines  `In order to track the pipelinerun progress run:
tkn pipelinerun logs build-and-deploy-run-z2rz8 -f -n pipelines-tutorial` will not be printed